docs/Migrating.md

```diff
@@ -1,5 +1,16 @@
 # Migrating
 
+## Migrating from ML-Agents toolkit v0.4 to v0.5
+
+### Unity API
+ * `unity-environment` has been renamed `MLAgentsSDK`.
+
+### Python API
+ * The `python` folder has been reorganized. We removed the `python` folder and replaced it with `ml-agents`. The folder `ml-agents` now contains the packages `mlagents.env` to interact directly with Unity environments and `mlagents.trainers` to train environments.
```

Comment:
Here you say that we both reorganized the `python` folder and removed it. Perhaps it is better to say at the beginning of this section that we reorganized the entire repo, and then list the changes.